.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Documentation:
```

Comment:
Does anything need to be changed for rubocop to run as a part of CI?

Reply:
Not really, I'm running it through coditsu.io now which post back status. This file is mostly for local development